src/rng.cpp

```diff
@@ -1,19 +1,30 @@
-#include "output.h"
 #include "rng.h"
-#include <stdlib.h>
+#include <random>
 
-long rng(long val1, long val2)
+namespace {
+std::mt19937 random_generator {std::random_device{}()};
```

Comment:
I know `brogue` and some other procedurally generated games use two or more generators. (1) for substantive randomness influencing the layout and state of the world. (2) for incidental randomness like for combat.